.github/actions/dotnet-solution-build-and-test/action.yml

```
@@ -54,13 +52,14 @@ runs:
       shell: pwsh
       run: |
         dotnet build ${{ inputs.SOLUTION_FILE_PATH }} --no-restore --configuration ${{ inputs.BUILD_CONFIGURATION }}
-
-    - name: Login to use Azure resources in integration tests
+    
+    - name: Az CLI login
```

I feel the old name was more informative, telling developers exactly why we login
```suggestion
    - name: Login to use Azure resources in integration tests
```